lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -192,10 +192,10 @@ def cmd_edit_tabs(str, words)
   end
 
   def cmd_reload_lib_help
-    print_line 'Usage: reload_lib lib/to/reload.rb [...]'
+    print_line 'Usage: reload_lib lib/to/reload.rb [...] [diff]'
```

Comment:
Make the usage more clear as this is a little confusing. It is really `reload_lib` file or diff, it doesn't appear both are an option at the same time.

Reply:
I don't really like this argument. And what if there's a file named `diff`? Perhaps you want to use `OptionParser`.

Reply:
It appears I already added a check for `.rb`.

Reply:
Yeah, adding an option `-a` and/or `--all` would be more semantic.

Reply:
Yeah, I like `-a`/`--all` much better, even if you don't use `OptionParser`.

Reply:
I think `OptionParser` would be a good extra effort as well, since this now has several options. You can look at the `repeat` and `grep` commands for examples.